h/formatters/annotation_hidden.py

```diff
@@ -59,11 +59,7 @@ def _current_user_is_author(self, annotation):
         return self._user and self._user.userid == annotation.userid
 
     def _is_hidden(self, annotation):
-        id_ = annotation.id
+        if annotation.id not in self._cache:
+            self._cache[annotation.id] = annotation.is_hidden
```

Comment:
The major change is this line which swaps `hidden = self._moderation_svc.hidden(annotation)` for `annotation.is_hidden`. After that I rewrote it a bit because it was really confusing me.